docs/ecctl_deployment_show.adoc

```diff
@@ -26,16 +26,17 @@ ecctl deployment show <deployment-id> [flags]
 === Options
 
 ----
-      --generate-update-payload   Outputs JSON which can be used as an argument for the --file flag with the update command.
-  -h, --help                      help for show
-      --kind string               Optional deployment resource kind (apm, appsearch, kibana, elasticsearch)
-  -m, --metadata                  Shows the deployment metadata
-      --plan-defaults             Shows the deployment plan defaults
-      --plan-history              Shows the deployment plan history
-      --plan-logs                 Shows the deployment plan logs
-      --plans                     Shows the deployment plans
-      --ref-id string             Optional deployment kind RefId, if not set, the RefId will be auto-discovered
-  -s, --settings                  Shows the deployment settings
+      --clear-transient transient   Removes the transient field in order to make read - edit - write loop safer. The default value of clear-transient depends on the value of `generate-update-payload`. If `generate-update-payload` is true `clear-transient` defaults to true. Otherwise defaults to false.
```

Comment:
Looks like some issue with the docs generator: It adds a transient that shouldn't be there `--clear-transient transient    Removes the`

It seems like it is due to `` in the text. Maybe some special syntax 🤔